src/coreclr/src/pal/src/misc/jitsupport.cpp

```diff
@@ -34,95 +34,95 @@ PAL_GetJitCpuCapabilityFlags(CORJIT_FLAGS *flags)
 // available, using the latest kernel for release should be sufficient.
 #ifdef HWCAP_AES
     if (hwCap & HWCAP_AES)
-        CPUCompileFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_HAS_ARM64_AES);
+        CPUCompileFlags.Set(InstructionSet_Aes);
```

Comment:
Do we want to keep `arm` somewhere in this name, so it can be differentiated from other platforms that have AES?

Reply:
In the C++ world, the InstructionSet_XXX enumeration values are architecture specific. In the C# world they are not. The Instruction_XXX names were specifically chosen to match the existing names as used in the jit to make the delta for this change reviewable.